orca-sql/src/main/kotlin/com/netflix/spinnaker/config/PipelineRefProperties.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2024 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Free Trial 1.0.0 license
```

Comment:
This is a different license than the Apache license we use elsewhere.  Mind changing it?

Reply:
I updated license on the new files